### DISCUSSION TITLE
Vercel's Dashboard Hover Style

### DISCUSSION THREAD

**Discussion by dpaulos6** [vercel/next.js#45382] [Help]
I've noticed Vercel's dashboard has a very cool styling when you hover an item. Basically, it has a very common background lighter color on hover, but the cool part is that if you hover another item, the background sort of slides to the item. Anyone could help me recreate this effect or give me any tips of how it was made?

**Comment by guitheengineer** [comment]
@dpaulos6 here's a blog post that recreated vercel's effect:
https://emilkowal.ski/writing/tabs

here's the codesandbox with the implementation:
https://codesandbox.io/s/crazy-hertz-nzmpj?file=/src/Tabs.js:1465-1531

**Comment by alyhxn** [comment]
If anyone wants a better and simpler solution, here you go
https://codepen.io/lvtgsfkc-the-scripter/pen/rNXaVOE
